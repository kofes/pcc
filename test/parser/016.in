var
ident1, ident2 : integer;
ident3, ident1 : char;
