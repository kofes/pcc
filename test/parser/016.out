Duplicate identifier "IDENT1" in pos (3, 9);
